### PR TITLE
Created Landing Page and Routed it to the Login page

### PR DESCRIPTION
**Pull request by basticks02**
Created the layout of the Login page, routed it to the Landing page. 
Bug: margin on the right of the page
Screenshot 2024-06-25 at 5 56 43 PM
Screenshot 2024-06-25 at 5 53 55 PM

### CODE REVIEW

File: frontend/src/Landing/Landing.jsx [hunk 1]
```diff
@@ -3,8 +3,53 @@ import './Landing.css'
 export default function Landing() {
   return (
     <>
-      <nav></nav>
+      <nav className='navbar'>
```
Comment (cherieliu) [inline frontend/src/Landing/Landing.jsx:6]: do we want to pull out the navigation bar and create a separate component for it - assuming the nav bar will appear on multiple different pages? 

with a component, we could probably also customize which tabs appear (i.e. patients and physicians will see different tabs or the logged out page might not have "My Profile")

File: frontend/src/Login/Login.jsx [hunk 2]
```diff
@@ -1,9 +1,41 @@
...
 export default function Login() {
   return (
-    <div>
+    <>
+      <div className='logincontainer'>
 
-    </div>
+        <div className='welcomestuff'>
+          <h1>Welcome Back</h1>
+          <p>Commited to serivce excellence</p>
```
Comment (cherieliu) [inline frontend/src/Login/Login.jsx:11]: nit: `service`

File: frontend/src/Login/Login.jsx [hunk 3]
```diff
@@ -1,9 +1,41 @@
...
+          <div className='socialmediaicons'>
+              <i className="fa-brands fa-facebook"></i>
+              <i className="fa-brands fa-twitter"></i>
+              <i className="fa-brands fa-instagram"></i>
+              <i className="fa-brands fa-youtube"></i>
+          </div>
```
Comment (cherieliu) [inline frontend/src/Login/Login.jsx:17]: super cool integration of these apps! are we going to link them to the clinic's accounts?

**Review by cherieliu — COMMENTED** [review]
great job!! the landing page and features look super cool :) overall, I just have a few questions/ideas to implement later on.

File: frontend/src/Login/Login.jsx (hunk 2 above)
Comment (m5matthew) [inline frontend/src/Login/Login.jsx:11]: typo here?

File: frontend/src/Landing/Landing.jsx (hunk 1 above)
Comment (basticks02) [reply frontend/src/Landing/Landing.jsx:6]: Never thought of that. Working on it!

File: frontend/src/Login/Login.jsx (hunk 3 above)
Comment (basticks02) [reply frontend/src/Login/Login.jsx:17]: Most likely, yes.

**Pull request merged**